action.yml

```diff
@@ -20,13 +20,15 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Install NPM deps
-      run: npm install
-      shell: bash
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '16.x'
     - name: Deploy
       env:
         HUBSPOT_PORTAL_ID: ${{ inputs.portal_id }}
         HUBSPOT_PERSONAL_ACCESS_KEY: ${{ inputs.personal_access_key }}
       run: |
-        npx @hubspot/cli@3.0.9 upload ${{ inputs.src_dir }} ${{ inputs.dest_dir }} --use-env 
+        npx --yes --package=@hubspot/cli@3.0.9 --call='hs upload ${{ inputs.src_dir }} ${{ inputs.dest_dir }} --use-env'
```

Comment:
- NPX now prompts the user to install the package, `--yes` skips that
- `--package` explicitly installs the latest version
- `--call` runs the actual CLI command